deepchem/models/autoencoder_models/autoencoder.py

```diff
@@ -0,0 +1,106 @@
+"""
+TODO(LESWING) Remove h5py dependency
+TODO(LESWING) Remove keras dependency and replace with functional keras API
+"""
+
+from deepchem.models import Model
+from deepchem.models.autoencoder_models.model import MoleculeVAE
+from deepchem.feat.one_hot import zinc_charset
+import os
+from subprocess import call
+
+
+class TensorflowMoleculeEncoder(Model):
+
+  def __init__(self,
+               model_dir=None,
+               weights_file="model.h5",
+               verbose=True,
+               charset_length=len(zinc_charset),
+               latent_rep_size=292):
+    """
+    :param model_dir: Folder to store cached weights
```

Comment:
Aren't we using numpy doc format for docstrings? If this is wrong I'll fix my other PRs later to reflect...